### DISCUSSION TITLE
Crash when opened and closed the "Run latency certifier"

### DISCUSSION THREAD

**Discussion by JeromTWinL** [ppy/osu#23546] [Q&A] [answered]
### Type
Crash to desktop
### Bug Description
As you saw in the [video](https://drive.google.com/file/d/1euJ99QX4TpStHelny61XyZGps2JC4bJ4/view?usp=drivesdk), after opening and closing the latency certifier, osu! started to lag (which was perfectly fine before opening it), the **clicks** were detected as *hold* (as it lagged) after few seconds, it managed to crash my phone, but the record was saved. After the black screen, my phone's `System UI` crashed, but I managed to fix it. Still not sure if my phone specs was the problem or not. (So I opened it as discussion and briefly explained it)
Here are my phone specs: (if needed)

```
CPU:Qualcomm Snapdragon SDM660
GPU:Adreno 512
RAM: 6GB
```
### Version
 2023.513.0 (Android)

### Logs
[network.log](https://github.com/ppy/osu/files/11471308/network.log)
[performance.log](https://github.com/ppy/osu/files/11471309/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11471310/runtime.log)

**Comment by bdach** [comment, marked as answer]
Escalated to issue #23547
